templates/paragraphs/paragraph--oe-content-row--variant-inpage-navigation.html.twig

```diff
@@ -0,0 +1,15 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Content row' #type and 'Inpage navigation' #variant paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{{ pattern('inpage_navigation', {
+  'title': title|default(''),
```

Comment:
The field can be empty so I would not add a default like in oe_theme, we don't have those requirements, that's good, but, what's the point of setting an empty default?
```suggestion
  'title': title,
```